CMakeLists.txt

```diff
@@ -40,6 +40,15 @@ if((CMAKE_SYSTEM_NAME STREQUAL "") AND (NOT CMAKE_HOST_SYSTEM_NAME STREQUAL "Lin
     set(UAGENT_P2P_PROFILE OFF)
 endif()
 
+###############################################################################
+# Dependencies
+###############################################################################
+set(FAST_RTPS_VERSION 1.8.1)
```

Comment:
I would use an internal cache variable to avoid corruption.

Reply:
I will change all internal variables to be prefixed with "_lower_case".